extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -121,25 +125,34 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 
 	GoogleTag.prototype.addSlot = function (adElement) {
 		var sizes = adElement.getSizes(),
-			slot = slots[adElement.getId()];
+			slot;
 
 		log(['addSlot', adElement], 'debug', logGroup);
 
 		adElement.setPageLevelParams(pageLevelParams);
+
+		if (sizes) {
+			slot = window.googletag.defineSlot(adElement.getSlotPath(), sizes, adElement.getId());
+		} else {
+			slot = window.googletag.defineOutOfPageSlot(adElement.getSlotPath(), adElement.getId());
+		}
+
 		if (!slot) {
-			if (sizes) {
-				slot = window.googletag.defineSlot(adElement.getSlotPath(), sizes, adElement.getId());
-			} else {
-				slot = window.googletag.defineOutOfPageSlot(adElement.getSlotPath(), adElement.getId());
-			}
-			slot.addService(pubAds);
-			window.googletag.display(adElement.getId());
-			slots[adElement.getId()] = slot;
+			console.log('bogna', 'problem with adding slot:', adElement.getSlotName(), window.googletag.getSlots());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2228)